pkg/machines/scripts/create_machine.sh

```diff
@@ -85,13 +89,21 @@ else
     fi
 fi
 
+
+if [ "$STORAGE_POOL" != "Automatic" ]; then
+    CHECK_PARAM="--check path_in_use=off"
```

Comment:
Why you need to pass this? I can imagine that you expect the volume you pass to always exist so you don''t need virt-install to do the check for you. But if this doesn't hurt to not be specified I 'd rather remove it.

Reply:
That's not what this parameter does. What it does it it checks if disk is already used by another VM. Default value (on) throws error if disk is already used. (see man virt-install)
However this PR allows user to also choose storage, which is used by another VM. So we want the parameter "--check path_in_use=off" in case "custom storage" was specified by user.

Reply:
Oh right, I got mislead.

Generally, I would suggest introducing the changes one by one.

The support for storage pool/volumes choice would make sense by itself, without allowing multiple VMs to use the same pool. You don't need to squash everything in a single commit. 

In addiction this --check path_in_use=off newly introduced option affects not only your newly introduced installation option choice, but all existing ones as well.

For example, if someone chooses 'Use existing disk option' now, he will not see any warning that some other VM is using that disk, (because this was never implemented) but also virt-install will not fail, because you removed the check here. This can lead in a user starting a new VM with a used disk, resulting in corruption in the disk, which will be mounted rw in two VMs.

Reply:
Is mentioned in IRC, --check path_in_use is set to off only for the functionality of this PR (that's what [ "$STORAGE_POOL" != "Automatic" ] does). Otherwise it will say on default value, therefore not affecting anything outside of this PR I believe.